src/flow.h

```diff
@@ -503,6 +537,9 @@ typedef struct Flow_
     uint32_t tosrcpktcnt;
     uint64_t todstbytecnt;
     uint64_t tosrcbytecnt;
+#if defined(ENABLE_ETM)
+    FlowSPLT splt;
```

Comment:
this is a large member, so it would be probably have to use the `FlowStorage` API when the feature is done as a runtime option.